src/config.js

```diff
@@ -7,5 +7,9 @@ module.exports = {
   baseURL: process.env.baseURL || 'http://localhost:8080',
   jwtSigningKey: new Buffer(jwtSigningKeyB64, 'base64'),
   sqlite: process.env.sqlite || './data/podverseWeb.sqlite',
-  databaseName: 'podverse'
+  databaseName: 'podverse',
+
+  // TODO: change the auth0 values to podverse.fm's clientId and domain before deployment
+  auth0ClientId: 'tTGQrl5CenMDdpzcKNpmLIginyRgBJNN',
```

Comment:
Notice how in other configuration I use `process.env.* || 'defaultValue`

You can try this approach out.. try 

```
auth0ClientId: process.env.auth0ClientId || 'whatever'
```

THEN when you start the server or use webpack or whatever run this command before

`export auth0ClientId='the-real-client-id'`

Essentially in deployment I like to have all configuration values settable via environment variables on the shell.

```
export baseURL='https://podverse.fm'
export sqlite='/full/path/to/podverse.sql'

# ...

node src/server.js
```